server/src/http/index.ts

```diff
@@ -0,0 +1,23 @@
+import * as bodyParser from 'body-parser';
+import * as cors from 'cors';
+import * as dotenv from 'dotenv';
+import * as express from 'express';
+import * as logger from 'morgan';
+
+import routes from './router';
+
+dotenv.config();
+
+const app: express.Application = express();
+const corsMiddleware = cors({ origin: '*', preflightContinue: true });
+
+app.use(bodyParser.urlencoded({ extended: false }));
+app.use(bodyParser.json());
+app.use(logger('dev'));
+app.use(corsMiddleware);
+app.options('*', corsMiddleware);
+routes(app).listen(process.env.PORT);
+
+export default app;
```

Comment:
Based on https://basarat.gitbooks.io/typescript/docs/tips/defaultIsBad.html , http://jypepin.com/anti-pattern-es6-default-exports and many more, default exports are considered harmful, maybe we should (maybe not in this PR) move away from it and prefer named exports.

Reply:
done